Examples/Python/python/acts/examples/simulation.py

```diff
@@ -594,7 +594,7 @@ def addGeant4(
     g4conf = geant4SimulationConfig(
         level=customLogLevel(),
         detector=g4detectorConstruction,
-        inputParticles="particles_input",
+        inputParticles="particles_selected",
```

Comment:
Actually, shouldn't it be the variable `particles_selected`, since this variable is populated correctly in the `if` clause in line 578?

Reply:
```suggestion
        inputParticles=particles_selected,
```

Reply:
Good catch !